bogenliga/src/app/modules/sportjahresplan/components/sportjahresplan/sportjahresplan.component.ts

```diff
@@ -232,9 +232,11 @@ export class SportjahresplanComponent extends CommonComponent implements OnInit
       tableContentRow.satzpunkte = match.satzpunkte;
       tableContentRow.version = match.version;
 
+
       this.tableContentMatch.push(tableContentRow);
     }
     this.matchRows = toTableRows(this.tableContentMatch);
+    this.tableContentMatch = [];
```

Comment:
Fände ich schöner, wenn Du das direkt oben in der Methode `onView` machen würdest (z.B. nach `this.visible=true`). Dann ist's sowohl im Fehlerfall als auch im Erfolgsfall zurückgesetzt.